doc/user_guide/getting_started.rst

```diff
@@ -236,32 +236,32 @@ so due to a couple of things that all contribute:
   data (most often for plotting, but applies for any other operation too), you
   want to keep it ordered for "fast access".
 
-- In python (more explicitly `numpy`) the "fast axes order" is the C-order.
-  This means that the **last** axis of a numpy array is fastest to iterate over
-  (i.e. the lines in the book). There are alternatives in other languages,
-  namely the F (for Fortran)-order, where it's the reverse - the first axis of
-  an array is the fastest to iterate over. Usually in all implementations
-  iterating over some axis in the middle is slow, as you have to "pick the same
-  line from all pages", or something similar.
+- In Python (more explicitly `numpy`) the "fast axes order" is C order (also
+  called row-major order). This means that the **last** axis of a numpy array
+  is fastest to iterate over (i.e. the lines in the book). There are
+  alternatives in other languages, namely F order (column-major), where it is
+  the reverse - the first axis of an array is the fastest to iterate over.
+  Usually in all implementations iterating over some axis in the middle is
+  slow, as you have to "pick the same line from all pages", or something
+  similar.
 
-- Most datasets are recorded in the C-order. Taking EDS / EELS map as an
+- Most datasets are recorded in C order. Taking an EDS / EELS map as an
```

Comment:
Have EDS, EDS and SPED being defined before? I think that for non-microscopy users it would be better to use a more general example that doesn't assume that the user is familiar with EM.